tests/suites/0_stateless/03_0003_select_group_by.sql

```diff
@@ -7,3 +7,6 @@ SELECT number%3 as c1 FROM numbers_mt(10) where number > 2 group by number order
 SELECT number%3 as c1 FROM numbers_mt(10) where number > 2 group by c1 order by c1;
 
 SELECT number%3 as c1, number as c2 FROM numbers_mt(10) where number > 2 group by c1 order by c1;
+SELECT number%3 as c1, number as c2, max(c2) FROM numbers_mt(10) where number > 2 group by c1 order by c1;
+SELECT 1 + sum(number) as d, d + 3 FROM numbers(10) group by number % 4 order by d;
+SELECT min(sum(number)) FROM numbers(10);
```

Comment:
This case is not group by.
It would be better to move it to '03_0001_select_aggregator.sql'